Microsoft.HealthVault.Fhir/Codes/HL7/ObservationCategoryCodes.cs

```diff
@@ -17,11 +17,11 @@ public static class ObservationCategoryCodes
     {
         public const string System = "http://hl7.org/fhir/observation-category";
```

Comment:
Yes, I am planning to do that.

---
In reply to: [123834340](https://github.com/Microsoft/healthvault-fhir-library/pull/3#discussion_r123834340) [](ancestors = 123834340)